content/learn/getting-started/node-with-typescript.pt.md

```diff
@@ -0,0 +1,129 @@
+---
+title: nodejs-with-typescript
+displayTitle: 'Node.js com a TypeScript'
+description: 'Descubra o porquê da TypeScript ser uma ferramenta impressionante e aprenda a usá-la sozinho.'
+authors: sbielenica, ovflowd, vaishnav-mk, nazarepiedady
+category: learn
+---
+
+## O Que é a TypeScript
+
+**[TypeScript](https://www.typescriptlang.org)** é uma linguagem de código-aberto na moda mantida e desenvolvida pela Microsoft. É amada e usada por muitos programadores de software em todo mundo.
+
+Basicamente, é um superconjunto de JavaScript que adiciona novas capacidades à linguagem. A adição mais notável é a definição de tipo estático, algo que não está presente na JavaScript simples. Graças aos tipos, é possível, por exemplo, declarar quais tipos de argumentos estamos esperando e o que é que é exatamente retornado em nossas funções ou qual é a forma exata do objeto que estamos a criar. A TypeScript é uma ferramenta realmente poderosa e abre um mundo novo de possibilidades nos projetos de JavaScript. Ela torna o nosso código mais seguro e robusto prevenindo muitos bugs antes mesmo do código ser moldado - ela captura problemas durante o desenvolvimento do código e se integra maravilhosamente com os editores de código como Visual Studio Code.
```

Comment:
```suggestion
Basicamente, é um superconjunto de JavaScript que adiciona novas capacidades à linguagem. A adição mais notável é a definição de tipo estático, algo que não está presente na JavaScript simples. Graças aos tipos, é possível, por exemplo, declarar quais tipos de argumentos estamos esperando e o que é exatamente retornado em nossas funções ou qual é a forma exata do objeto que criamos. A TypeScript é uma ferramenta realmente poderosa e abre um mundo novo de possibilidades nos projetos de JavaScript. Ela torna o nosso código mais seguro e robusto prevenindo muitos bugs antes mesmo do código ser moldado - ela captura problemas durante o desenvolvimento do código e se integra maravilhosamente com os editores de código como Visual Studio Code.
```

Reply:
I do not see the necessity of changing it, and since there are no grammar or syntactical errors, it should remain as it is.